x-pack/plugins/security_solution/public/common/components/exceptions/viewer/index.tsx

```diff
@@ -374,6 +386,7 @@ const ExceptionsViewerComponent = ({
         />
 
         <ExceptionsViewerItems
+          disableActions={!canUserCRUD || !hasIndexWrite}
```

Comment:
Not for this PR but it would be cool if we could clarify our privilege reducers so that we don't have to do extra logic for what we need on each page. Looking through the code there's a lot of similar mixing of booleans to determine if the user has privileges to do an action / view something and it would be great if the reducers encapsulated that logic. Might mean we have to update the HTTP endpoints to return these booleans but that could be beneficial for ensuring we consistently enforce privileges against all components that need it. With the added benefit of receving updates to those privileges without having to make manual changes to each component.

Something similar to what was referenced in this issue: https://github.com/elastic/kibana/issues/80601 but more generic.